CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/dataaccess-category-preferences.md

```diff
@@ -0,0 +1,14 @@
+# Data Access View Category Preferences
```

Comment:
Are you able to rename this to "Data Access View" since that is the name of the tab in preferences